is_all_unique_elements.js

```diff
@@ -0,0 +1,16 @@
+
+const isAllUniqueElements = (array) => {
+  let isAllUnique = true;
```

Comment:
We may want to assign is AllUnique to something else. If the array is empty, it will evaluate to true.